raw-data/UW/uw_inventories.tsv

```diff
@@ -53,3 +53,43 @@ roseslabrada2015mako.pdf		Mako		i		i
 						n	"/l/ can also be variably nasalized when followed by nasal vowels"	
 				w		w		
 				ˀw		ˀw		
+
+berry1999abun.pdf	abun1252	Abun		p		p		
+				b		b		
+				t		t		
+				d		d		
+				k		k		
+				ɡ		ɡ		
+				d̠ʒ	"in the Abun Tat dialect to the east there is no separate /j/"	d̠ʒ		
+				mb		mb		
+				nd		nd		
+				n̠d̠ʒ	"Neither does the /nj/ phoneme occur in the Abun Tat dialect."	n̠d̠ʒ		
+				ŋɡ		ŋɡ		
+				f		f		
+				s		s		
+				ʃ		ʃ		
+						t̠ʃ	"A voiceless palatal affricate [tš] occurs in the Abun Ji dialect on occasion in free variation with the palatal fricative [š]"	
+						s	"/š/... neutralises to /s/ syllable-finally"	
+				m		m		
+				n		n		
+				ɲ		ɲ		
+						n̠d̠ʒ	"Neither does the /nj/ phoneme occur in the Abun Tat dialect. Rather it is an allophone of /ñ/"	
+						n	"/ñ/... neutralises to /n/ word-finally"	
+				w		w		
+				r		r		
+				j		j		
+						d̠ʒ	"in the Abun Tat dialect... [j] is in free variation with [y]"	
+				i		i	"For the high vowels, /i/ and /u/, tensing is particularly noticeable in open syllables"	
+						ɪ		
+				e		e	"For the mid vowels, /e/ and /o/, tensing is more noticeable when tone is high"	
+						ɛ		
+				u		u	"For the high vowels, /i/ and /u/, tensing is particularly noticeable in open syllables"	
+						ʊ		
+				o		o	"For the mid vowels, /e/ and /o/, tensing is more noticeable when tone is high"	
+						ɔ		
+				ɑ		ɑ		
+				˦	high/rising	˦		
+						˩˥	"When a low or falling tone precedes a high tone, the high tone begins with an upward pitch, like a rising tone, whereas elsewhere it is a high tone."	
```

Comment:
similar comment as for the low falling allophone: I'd put this at 3-5 (˧˥) rather than 1-5